docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,35 +3,36 @@ title: Visual Studio Tools for Docker on Windows
 description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 08/22/2018
+ms.date: 09/04/2018
 ---
 # Use Visual Studio Tools for Docker (Visual Studio on Windows)
 
-The developer workflow when using Visual Studio Tools for Docker is similar to the workflow when using Visual Studio Code and Docker CLI (it is based on the same Docker CLI). Visual Studio Tools for Docker makes it even easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. It's also able to execute and debug your containers via simple actions like **F5** and **Ctrl**+**F5**. You can also run and debug a group of containers (a whole solution) at the same time if they're defined in the same *docker-compose.yml* file at the solution level.
+The Visual Studio Tools for Docker developer workflow is similar to using Visual Studio Code and Docker CLI (it is based on the same Docker CLI). Visual Studio Tools for Docker makes it even easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. Execute and debug your containers via simple actions like **F5** and **Ctrl**+**F5**. You can also run and debug a group of containers (a whole solution) at the same time if they're defined in the same *docker-compose.yml* file at the solution level.
```

Comment:
I think the last sentence here about docker-compose.yml should be removed or put into a separate paragraph that deals specifically with the less-common "Container Orchestratration" option. It's confusing to see it mentioned here for users that don't choose that more complex option.